var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -370,6 +370,16 @@ def _microarch_target_args(self):
             # case can go away.
             args.append("TARGET=" + "RISCV64_GENERIC")
 
+        elif self.spec.satisfies("@0.3.19: target=a64fx"):
+            # Special case for Fujitsu's A64FX
+            if self.spec.satisfies("%gcc@11:"):
+                args.append("TARGET=A64FX")
+            elif self.spec.satisfies("%clang") or self.spec.satisfies("%fj"):
```

Comment:
Why this go together with the gcc branch?

Reply:
I can put it all together. I put it in separate branches because the second conditional comes from the `ISCLANG` Makefile variable.